src/main/java/com/listywave/common/config/SpringEnvironmentUtil.java

```diff
@@ -0,0 +1,30 @@
+package com.listywave.common.config;
+
+import com.listywave.common.constants.EnvironmentConstants;
+import java.util.Arrays;
+import java.util.stream.Stream;
+import lombok.RequiredArgsConstructor;
+import org.springframework.core.env.Environment;
+import org.springframework.stereotype.Component;
+
+@Component
+@RequiredArgsConstructor
+public class SpringEnvironmentUtil {
+
+    private final Environment environment;
+
+    public String getCurrentProfile() {
+        return getActiveProfile()
+                .filter(
+                        profile ->
+                                profile.equals(EnvironmentConstants.LOCAL.getValue())
+                                        || profile.equals(EnvironmentConstants.DEV.getValue())
+                )
+                .findFirst()
+                .orElse(EnvironmentConstants.LOCAL.getValue());
+    }
+
+    public Stream<String> getActiveProfile() {
+        return Arrays.stream(environment.getActiveProfiles());
+    }
+}
```

Comment:
```suggestion
@Component
@RequiredArgsConstructor
public class SpringEnvironmentUtil {

    private static final String LOCAL = "local";
    private static final String DEV = "dev";

    private final Environment environment;

    public String getCurrentProfile() {
        return Arrays.stream(environment.getActiveProfiles())
                .filter(this::isActivateDev)
                .findFirst()
                .orElse(LOCAL);
    }

    private boolean isActivateDev(String profile) {
        return profile.equals(DEV);
    }
}
```

이런 식으로 짜면 `EnvironmentConstants`도 제거될 수 있을 것 같고, 가독성이 더 좋아질 것 같아요.
추후 Spring Profile에 `prod`가 추가된다면 `isActiveDev`메서드에 `prod`를 추가하는 식으로 변경하기도 용이할 듯 합니다.

Reply:
좋습니다 ㅎㅎ